src/nbl/asset/utils/CIESProfile.cpp

```diff
@@ -200,7 +188,9 @@ core::smart_refctd_ptr<asset::ICPUImageView> CIESProfile::createIESTexture(Execu
     viewParams.subresourceRange.layerCount = 1u;
 
     auto imageView = ICPUImageView::create(std::move(viewParams));
-    bundle = asset::SAssetBundle(nullptr, { core::smart_refctd_ptr(imageView) });
+    const auto aHash = key + "?flatten=" + std::to_string(flatten);
+    asset::IAssetLoader::SAssetLoadContext ctx = { {}, nullptr };
+    SAssetBundle bundle = asset::SAssetBundle(nullptr, { core::smart_refctd_ptr(imageView) });
     aOverride->insertAssetIntoCache(bundle, aHash, ctx, 0u);
```

Comment:
also don't insert, caller inserts!

don't want the `asset::IAssetLoader::IAssetLoaderOverride* aOverride, const std::string key` parameters here at all

Reply:
I think we agreed the utility should do it, however resolved in following commit